docs/src/advanced/symmetries.md

```diff
@@ -3,41 +3,41 @@
 In this discussion we will only describe the situation for a monoatomic crystal
 ``\mathcal C \subset \mathbb R^3``, the extension being easy.
 A symmetry of the crystal is a real-space unitary matrix ``\tilde{S}``
-and a real-space vector ``\tilde{τ}`` such that
+and a real-space vector ``\tilde{\tau}`` such that
```

Comment:
why?

Reply:
I thought unicode didn't work, but actually does, it's pretty neat

Reply:
I'm leaving the latex in case we want to copy/paste it into a latex

Reply:
ok, but I'm not going to guarantee I'm going to stick to that.